.github/workflows/ci.yml

```
@@ -24,61 +24,71 @@ jobs:
-            coverage: false
+            coverage: true
             features: strict-compile-time-validation
             with-default-features: false
+            tools: false
     steps:
       - uses: actions/checkout@v5
       - name: Setup Rust
         uses: leynos/shared-actions/.github/actions/setup-rust@ebbaaa0d7a776ec3187f3940667e5c7c9379cdf5
       - name: Install bun
+        if: ${{ matrix.tools }}
         uses: oven-sh/setup-bun@735343b667d3e6f658f44d0eca948eb6282f2b76 # v2
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)